src/services/lomakkeet/backendMappings/index.js

```diff
@@ -1,16 +1,17 @@
 import { getAnchorPart } from "../../../utils/common";
 import { perustelut } from "./all-mappings";
-
+import * as R from "ramda";
 /**
  * Function tries to return an object filled with form data for backend.
  * @param {array} changeObjects - Array of change objects
  */
-export function fillForBackend(changeObjects) {
+export function fillForBackend(changeObjects, anchor) {
   if (changeObjects.length) {
     const head = getAnchorPart(changeObjects[0].anchor, 0);
+    const anchorInit = anchor && R.slice(0, R.lastIndexOf(".", anchor), anchor);
```

Comment:
We have utility functions for operations like this.

Reply:
But not for this case.